addons/mail/static/src/core/common/core.scss

```diff
@@ -80,6 +80,10 @@ a.o-discuss-mention {
     height: calc(100% - #{$o-navbar-height}); // no bottom-0 otherwise performance issue
 }
 
+.o-mail-hoverable:hover {
+    filter: brightness(98%);
+}
+
```

Comment:
I get we want to match style of hoverable of all discuss, but I think this is too soon to add this classname for this case and assume all occurrences will use same styling as this one, as a `filter: brightness(98%)`.

Better postpone this desire in a future PR. This put unnecessary pressure to fix the issue now when there's things that have genuinely more priority